UnityProject/Assets/Scripts/Systems/Chemistry/Reaction.cs

```diff
@@ -43,18 +46,18 @@ public bool IsReactionValid(ReagentMix reagentMix)
 				return false;
 			}
 
-			var reactionAmount = GetReactionAmount(reagentMix);
+			var reactionMultiple = GetReactionAmount(reagentMix);
 
 			if (useExactAmounts)
 			{
-				reactionAmount = (float)Math.Floor(reactionAmount);
-				if (reactionAmount == 0)
+				reactionMultiple = (float)Math.Floor(reactionMultiple);
+				if (reactionMultiple == 0)
```

Comment:
```suggestion
				if (reactionMultiple.Approx(0))
```
we did an extension for it. How many times do we have to teach you this lesson, old man?

![oldman](https://github.com/unitystation/unitystation/assets/43683714/f303ec36-2dc6-4568-a6bb-e1d5e6d4fd43)

Reply:
"
One or more cyclic dependencies detected between assemblies: Assets/Scripts/Objects/Research/Artifacts/Artifacts.asmdef, Assets/Assets.asmdef, Assets/Scripts/Systems/Chemistry/Editor/Chemistry.Editor.asmdef, Assets/Scripts/Systems/Chemistry/Effects/Chemistry.Effects.asmdef, Assets/Scripts/Systems/Chemistry/Chemistry.asmdef, Assets/Scripts/Core/Editor/Editor.asmdef, Assets/Tests/Tests.asmdef, Assets/Scripts/Tilemaps/Editor/TilemapsEditor.asmdef, Assets/Tests/PlayModeTests/play mode tests.asmdef
"

Reply:
where did we write the extension?

Reply:
In utilits

Reply:
And under what asmdef is that?

Reply:
looks like it's in assets